C/services/notification/notification.cpp

```diff
@@ -58,6 +58,19 @@ static void signalHandler(int signal)
 	}
 }
 
+// Displays service information in JSON format
+static void printServiceInfoAsJSON()
+{
+	std::cout
+		<< "{\"name\": " << "\"Notification Service\"" << ","
+		<< "\"description\": " << "\"Fledge Notification Service to send notifications\"" << ", "
+		<< "\"package\": " << "\"fledge-service-notification\"" << ", "
+		<< "\"type\": " << "\"notification\"" << ", "
+		<< "\"process_name\": " << "\"notification_c\"" << ", "
+		<< "\"process_script\": " << "\"services/notification_c\"}"
```

Comment:
@MarkRiddoch  From an API perspective, if service info is the only source of service details, then no transformation is needed at the API level. That means for process_script, the value should be `["services/notification_c"]`, since this is the expected format for insertion into the scheduled_processes table.
```
select * from scheduled_processes;
name|script|
notification_c|["services/notification_c"]
```

Reply:
This format has been used from the start, and the scheduler expects it. The only concern is whether the service must follow this format, or if the API should handle formatting internally.
When adding a new service, we need to ensure the payload is prepared in this expected format.